Flick/Controllers/ProfileViewController.swift

```diff
@@ -137,21 +130,41 @@ class ProfileViewController: UIViewController {
     }
 
     private func getCurrentUser() {
+        // Get profile of current user
         NetworkManager.getUserProfile { [weak self] userProfile in
             guard let self = self, let userProfile = userProfile else { return }
             DispatchQueue.main.async {
                 self.updateUserInfo(user: userProfile)
             }
         }
+
+        // Get friends of current user
+        NetworkManager.getFriends { [weak self] friends in
+            guard let self = self, !friends.isEmpty else { return }
+            self.friends = friends
+            DispatchQueue.main.async {
+                self.listsTableView.reloadSections(IndexSet([0]), with: .none)
+            }
+        }
     }
 
     private func getUser(userId: Int) {
+        // Get profile of another user
         NetworkManager.getUser(userId: userId) { [weak self] user in
             guard let self = self else { return }
             DispatchQueue.main.async {
                 self.updateUserInfo(user: user)
             }
         }
+
+        // Get friends of another user
+        NetworkManager.getFriendsOfUser(userId: userId) { [weak self] friends in
```

Comment:
Lol I feel like backend should just make getting friends same for any user... but for now this is fine haha